servers/rendering/shader_language.cpp

```diff
@@ -4111,18 +4119,24 @@ PropertyInfo ShaderLanguage::uniform_to_property_info(const ShaderNode::Uniform
 		case ShaderLanguage::TYPE_INT: {
 			if (p_uniform.array_size > 0) {
 				pi.type = Variant::PACKED_INT32_ARRAY;
+				// TODO: Handle range and encoding for for unsigned values.
```

Comment:
There should probably be some hint for the editor to treat `PACKED_INT32_ARRAY` as unsigned for the purposes of editing, as a future improvement